pkg/build/build.go

```diff
@@ -25,6 +26,17 @@ func BuildContainerDisk(imgPath string, checksum string) (v1.Image, error) {
 		return nil, fmt.Errorf("error appending the image layer: %v", err)
 	}
 
+	cf, err := img.ConfigFile()
+	if err != nil {
+		return nil, fmt.Errorf("error getting the image config file: %v", err)
+	}
+
+	cf.Architecture = ImageArchitecture
```

Comment:
Woult it make sense to set the label also here with `cf.Labels` and only mutate the image config once?

Reply:
Good call :+1: